SignInCheckIn/SignInCheckIn/Services/EventService.cs

```diff
@@ -80,6 +80,33 @@ public EventDto GetCurrentEventForSite(int siteId)
         //    return Mapper.Map<MpEventDto, EventDto>(currentEvents.First());
         //}
 
```

Comment:
so we are bringing it back from the dead as it wasnt being used after we moved that stuff to the stored proc